command-parser.js

```diff
@@ -133,6 +135,237 @@ function canTalk(user, room, connection, message, targetUser) {
 	return true;
 }
 
+var CommandContext = exports.Context = (function () {
```

Comment:
`CommandContext = exports.Context`

I'd rather simply call it `Context` or `CommandContext` everywhere.